include/mockturtle/views/cnf_view.hpp

```diff
@@ -53,6 +53,94 @@ struct cnf_view_params
 {
   /*! \brief Write DIMACS file, whenever solve is called. */
   std::optional<std::string> write_dimacs{};
+  bool auto_update{true}; /* only used when AllowModify = true */
```

Comment:
Is this a parameter that should be set by the user? If so, it should be documented (similar to `write_dimacs`)